<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AthleteDialog</class>
 <widget class="QDialog" name="AthleteDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>266</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>TurnFix - Teilnehmerdaten</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/TurnFix.qrc">
    <normaloff>:/appicons/icon.png</normaloff>:/appicons/icon.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="1" colspan="2">
       <widget class="QComboBox" name="cmb_sex"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="genderLabel">
        <property name="text">
         <string>Geschlecht:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QCheckBox" name="chk_dat">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>exakt</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="yobLabel">
        <property name="text">
         <string>Geburtsjahr:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDateEdit" name="dae_year">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>85</width>
          <height>0</height>
         </size>
        </property>
        <property name="displayFormat">
         <string>yyyy</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="2">
       <widget class="QLineEdit" name="txt_id">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="inputMask">
         <string>000000000000000000</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="firstnameLabel">
        <property name="text">
         <string>Vorname:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="clubLabel">
        <property name="text">
         <string>Verein:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="firstNameInput"/>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QComboBox" name="cmb_club">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="licenseLabel">
        <property name="text">
         <string>Startpass-Nr.:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lastnamneLabel">
        <property name="text">
         <string>Nachname:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="lastnameInput"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="but_save">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>speichern</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>firstNameInput</tabstop>
  <tabstop>lastnameInput</tabstop>
  <tabstop>dae_year</tabstop>
  <tabstop>chk_dat</tabstop>
  <tabstop>cmb_sex</tabstop>
  <tabstop>cmb_club</tabstop>
  <tabstop>txt_id</tabstop>
  <tabstop>but_save</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/TurnFix.qrc"/>
 </resources>
 <connections/>
</ui>
